firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -391,21 +390,13 @@ private void sendSessionLog(String name, Timer startTime, Timer endTime) {
   }
 
   /**
-   * Only send screen trace if FrameMetricsAggregator exists and the activity is hardware
-   * accelerated.
+   * Only send screen trace if FrameMetricsAggregator exists.
```

Comment:
In the PR description you mention that for non hw accelerated views this will be 0. Should the javadoc mention that anywhere? Not here, since this is a private method.

Reply:
Thank! Added a comment to `sendScreenTrace(Activity activity)`. There is no public method about screen traces because screen traces are measured automatically by Fireperf and users cannot measure screen traces themselves.